lib/db/schema/patch-015-016.sql

```diff
@@ -0,0 +1,46 @@
+-- Add queries for metrics reporting
+
+CREATE PROCEDURE `countAccountsCreatedBefore_1` (
+    IN createdBefore BIGINT UNSIGNED
+)
+BEGIN
+    SELECT COUNT(*) AS count FROM accounts
+    WHERE createdAt < createdBefore;
+END;
+
+CREATE PROCEDURE `countVerifiedAccountsCreatedBefore_1` (
+    IN createdBefore BIGINT UNSIGNED
+)
+BEGIN
+    SELECT COUNT(*) AS count FROM accounts
+    WHERE createdAt < createdBefore
+    AND emailVerified = true;
+END;
+
+CREATE PROCEDURE `countMultiDeviceAccounts_1` ()
+BEGIN
+    SELECT COUNT(*) AS count FROM (
+        SELECT uid FROM sessionTokens
+        GROUP BY uid
+        HAVING COUNT(tokenId) > 1
```

Comment:
I accidentally this out of my description in #72,  but what do you think about filtering for "active sessions" in these queries, using the newly-added lastAccessTime?  For example, rather than "number of accounts with multiple sessions" it could be "number of accounts with multiple sessions active in the last 30 days".

ISTM this will give us a little bit of a guard against counting e.g. transient web login sessions that were never properly cleaned up and aren't active any more.

/cc @billmaggs for perspective on whether this makes sense from a product perspective.